### PR TITLE
Android splash screen

### PR DESCRIPTION
**Pull request by Dallanosm**
Added Android Root classes and Splash Screen.

### CODE REVIEW

File: android/app/src/main/java/com/votlin/android/extensions/ViewExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import android.view.View
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+/**
+ * ViewExtensions.
+ */
+fun ImageView.load(url: String) {
+    Glide.with(this.context)
```
Comment (sergiocasero) [inline android/app/src/main/java/com/votlin/android/extensions/ViewExtensions.kt]: Glide works with ImageView as with() parameter, we don't need the context :)

File: android/app/src/main/java/com/votlin/android/ui/SplashActivity.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import com.votlin.client.presentation.SplashPresenter
+import com.votlin.client.presentation.SplashView
+
+class SplashActivity: RootActivity<SplashView>(), SplashView {
+
+    override val presenter: SplashPresenter by instance()
+
+    override val layoutResourceId: Int = R.layout.activity_splash
+
+    override val activityModule: Kodein.Module = Kodein.Module{
```
Comment (sergiocasero) [inline android/app/src/main/java/com/votlin/android/ui/SplashActivity.kt]: code style... Module{ <- hahaha same for next line)

File: android/app/src/main/java/com/votlin/android/ui/TalksActivity.kt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+class TalksActivity : RootActivity<TalksView>(),TalksView {
+
+    companion object {
+        fun getCallingIntent(context: Context) = Intent(context, TalksActivity::class.java)
+    }
+
+    override val presenter: TalksPresenter by instance()
+
+   override val layoutResourceId: Int = R.layout.activity_talks
+    override val activityModule: Kodein.Module = Kodein.Module{
```
Comment (sergiocasero) [inline android/app/src/main/java/com/votlin/android/ui/TalksActivity.kt:25]: Codestyle.. haha

**Review by sergiocasero — CHANGES_REQUESTED** [review]
Please, take a look to the comments :)

File: android/app/src/main/java/com/votlin/android/extensions/ViewExtensions.kt (hunk 1 above)
Comment (Dallanosm) [reply android/app/src/main/java/com/votlin/android/extensions/ViewExtensions.kt]: Nope, our current version of Glide doesn't have with() :(

**Pull request merged**